ShaiRandom/Generators/IEnhancedRandomExtensions.cs

```diff
@@ -321,7 +321,7 @@ public static T RandomElement<T>(this IEnhancedRandom rng, ReadOnlySpan<T> items
         {
             if (maxTries <= 0)
                 throw new ArgumentOutOfRangeException(nameof(maxTries),
-                    "Value must >= 0; for infinite retries, use the overload without a maxTries parameter.");
+                    "Value must > 0; for infinite retries, use the overload without a maxTries parameter.");
```

Comment:
Should this be `must be > 0` ? I'm not sure if .NET uses `>` as a verb; if it does, this is fine.